Orange/widgets/data/owsql.py

```diff
@@ -55,8 +55,8 @@ class Outputs:
     port = Setting(None)
     database = Setting(None)
     schema = Setting(None)
-    username = ""
-    password = ""
+    username = Setting(None)
```

Comment:
What about adding a migrate_settings (class) method and storing the username and password to credentials manager there?